elliottlib/cli/find_bugs_cli.py

```diff
@@ -226,8 +225,13 @@ def find_bugs_cli(runtime: Runtime, advisory, default_advisory_type, mode, check
             sweep_cutoff_timestamp = bzutil.approximate_cutoff_timestamp(runtime.assembly_basis_event, brew_api, runtime.rpm_metas() + runtime.image_metas())
 
         if sweep_cutoff_timestamp:
-            green_print(f"Filtering bugs that have changed to one of the desired statuses before the cutoff time {datetime.utcfromtimestamp(sweep_cutoff_timestamp)}...")
-            qualified_bugs = bzutil.filter_bugs_by_cutoff_event(bzapi, bugs, status, sweep_cutoff_timestamp)
+            green_print(f"Filtering bugs that have changed ({len(bugs)}) to one of the desired statuses before the "
+                        f"cutoff time"
+                        f" {datetime.utcfromtimestamp(sweep_cutoff_timestamp)}...")
+            qualified_bugs = []
+            for chunk_of_bugs in chunk(bugs, 500):
```

Comment:
I always prefer explicit module level "constants" insted of hardcoded numeric values that will be hard to maintain in the future